R/reorder.R

```diff
@@ -134,10 +134,23 @@ fct_inorder <- function(f, ordered = NA) {
 
 #' @export
 #' @rdname fct_inorder
-fct_infreq <- function(f, ordered = NA) {
+#' @param weight Optional vector of numeric values to weight by.
+fct_infreq <- function(f, weight = NULL, ordered = NA) {
```

Comment:
Do you expect the argument ordering here to matter? Probably not much? i.e. if someone specified `ordered` without naming the argument. Seems unlikely

Reply:
Yeah, I think that seems ok